sdks/go/test/integration/synthetic/synthetic_test.go

```diff
@@ -15,6 +15,14 @@
 
 package synthetic
 
+// beam-playground:
+//   name: SyntheticTest
+//   description: Unit-test for the Synthetic example.
+//   multifile: false
+//   pipeline_options:
+//   categories:
+//	   - Testing
+
```

Comment:
Seems like this test is something internal, let's remove it. It might be not so interesting for Playground users